hydro_deploy/core/src/deployment.rs

```diff
@@ -31,7 +31,7 @@ impl Deployment {
         self.add_host(LocalhostHost::new)
     }
 
-    #[allow(non_snake_case)]
+    #[allow(non_snake_case, clippy::too_many_arguments)] // TODO(mingwei)
```

Comment:
Yeah, we should replace this with a builder like for `HydroflowCrate`.

Reply:
Will address separately #1364